src/site/assets/js/execute-axe-check.js

```diff
@@ -0,0 +1,78 @@
+function processAxeCheckResults(error, results) {
+  if (error) {
+    console.log('Error executing the Axe check!');
+    console.log(error);
+    return;
+  } else if (results.violations.length === 0) {
+    return;
+  }
+
+  var bannerEl = document.createElement('div');
+
+  var details = '<details class="vads-u-background-color--gold-lighter vads-u-padding--1">';
+  details += '<summary>There are (' + results.violations.length + ') accessibility issues on this page.</summary>';
```

Comment:
I'd like to add a few classes and an H4 to this block to adjust the background color and offer some more visual contrast:

```javascript
var details = '<details class="vads-u-background-color--primary-alt-lightest  vads-u-padding--1 vads-u-border-color--primary-darker vads-u-border-bottom--2px">';
details += '<summary><h4 class="vads-u-margin-y--2 vads-u-display--inline-block">There are (' + results.violations.length + ') accessibility issues on this page.</h4></summary>';
```